src/operations/secrets.md

```diff
@@ -0,0 +1,86 @@
+
+# Introduction
+The Toronto Community Network uses [Bitwarden](https://bitwarden.com/) to share and manage secrets across the organization.
+
+Bitwarden is a free and open-source password management service that stores sensitive information (such as website credentials) in an encrypted vault.
+
+Bitwarden supports the following clients:
+
+- Desktop (Linux/MacOS/Windows)
+- Web Browser (Chrome/Safari/Firefox/Vivaldi/Opera/Brave/Edge/Tor Browser)
+- Mobile (Android/iPhone)
+- Command Line (via NPM)
+
+For more information about supported clients, see Bitwarden's [download options](https://bitwarden.com/download/).
+
+
+# Accessing Secrets
+The Toronto Community Network is using the self-hosted version of Bitwarden and URL can be accessed at: [https://tomesh.infectedroot.com](https://tomesh.infectedroot.com).
+
+For secrets to be shared, they must exist within the [Toronto Community Network organization](https://tomesh.infectedroot.com/#/organizations/e4573938-547f-4157-a977-b28fc15bcec0/vault).
+
+When using the Bitwarden applications, please make sure to [change the Server URL](https://bitwarden.com/help/article/change-client-environment/#:~:text=On%20the%20home%20screen%20of,com%20for%20the%20Server%20URL) in the Bitwarden settings to the URL [https://tomesh.infectedroot.com](https://tomesh.infectedroot.com).  
+
+# Obtaining Access
+Obtaining access to Bitwarden is a 4-step process.
+ 
+1.  Send an email to operations@tomesh.net with the following information:
+	- Your Name
+	- GitHub Handle
+	- Detailed descripition of your use case or requirement.
+
+2.  A member of our Project Operations team will invite you to Bitwarden.
+	- Bitwarden will send you an email from tomeshnet@gmail.com
+	- Please make sure to whitelist the email address.
+  
+3.  Use the link in the invitation to create an account.
+
+4.  After registration, Bitwarden requires us to confirm your account. Upon confirmation, you will have access to the organization.
+
+We recommend that you setup [two-factor authentication](https://bitwarden.com/help/article/setup-two-step-login/) on your account to increase the security of your account. 
+
+
+# Access Control
+When storing credentials, you must store them within the [Toronto Community Network organization](https://tomesh.infectedroot.com/#/organizations/e4573938-547f-4157-a977-b28fc15bcec0/vault). Bitwarden organizes secrets in groups called [Collections](https://bitwarden.com/help/article/collections/) and within those [Collections](https://bitwarden.com/help/article/collections/) are four [User Types]( https://bitwarden.com/help/article/user-types-access-control/#user-types): _User_, _Manager_, _Admin_, and _Owner_.
+
+In order to maintain integrity of the Toronto Community Network operations, most users will be granted only  User or Manager access. The Project Operations team will determine your access based on your use case or requirements.
```

Comment:
```suggestion
In order to maintain integrity of the Toronto Community Network operations, most users will be granted only User or Manager access. The Project Operations team will determine your access based on your use case or requirements.
```

Double space, not sure if it will be rendered or not, but removing just in case.